Backend/database/migrations/0001_01_01_000000_create_users_table.php

```diff
@@ -15,30 +15,15 @@ public function up(): void
             $table->string("email")->unique();
             $table->timestamp("email_verified_at")->nullable();
             $table->string("password");
+            $table->string("role")->default("user");
```

Comment:
It's just a suggestion, but if we have only two roles, user and admin, maybe instead of typing the user role in users table, we could add policies to the project? Then specific actions would be reserved for the admin only.

Something like this:

UserPolicy:
```public function before(User $user)
    {
        if ($user->is_admin) {
            return true;
        }
    }

    public function access(User $user)
    {
        return Response::deny('Permission denied');
    }
    
    and next in the controller if u want to check the user's persmissions:

if (JWTAuth::user()->cannot('access', User::class)) {
            return response()->json(['message' => 'Permission denied'], 403);
        }

Reply:
ok, I've never done this before, but I can try